pysat_requirements.txt

```diff
@@ -0,0 +1,8 @@
+dask
+netCDF4
+numpy>=1.12
+pandas>=0.23
+portalocker
+scipy
+toolz
+xarray
```

Comment:
This file is required for the test since pysat requirements are not in the test pypi repository.  Can remove once pysat changes are merged